src/s3path.jl

```diff
@@ -342,14 +342,14 @@ Base.read(fp::S3Path) = Vector{UInt8}(s3_get(fp.config, fp.bucket, fp.key))
 
 Base.write(fp::S3Path, content::String; kwargs...) = Base.write(fp, Vector{UInt8}(content); kwargs...)
 
-function Base.write(fp::S3Path, content::Vector{UInt8}; part_size_mb=50, multipart::Bool=false, other_kwargs...)
+function Base.write(fp::S3Path, content::Vector{UInt8}; part_size_mb=50, multipart::Bool=true, other_kwargs...)
```

Comment:
Is there a reason to force the multi-part by default? I feel like there's probably more overhead to making that call vs. a single one if our content isn't too large.

Reply:
it seems with the check a few lines later that even if `multipart` is true here, if `length(content) < MAX_HTTP_BYTES` holds then we still use just a `s3_put`. So my read of it is it only changes the default for large files, since small files use `s3_put` anyway.